### DISCUSSION TITLE
saf-training/courses/user/

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#1] [Ideas]
# saf-training/courses/user/

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/

**Comment by aaronlippold** [comment]
update the above graphic

**Comment by karikarshivani** [comment]
It might help to turn the questions in section 1.2 into a poll with rating options instead of requiring long responses right at the beginning of the class. For example:
What changes would improve software assessment in your context?
- Automation
- Process improvement
- Hardening
- Continuous Monitoring
- Centralized Reporting
- Team Training

Students can of course expand on their responses in the chat or verbally.
